exams/tasks_test.py

```diff
@@ -338,22 +340,21 @@ def test_update_exam_run(self, update_mock):
         update_mock.assert_called_once_with(exam_run)
 
     @data(True, False)
-    @patch('exams.api.bulk_authorize_for_exam_run')
-    def test_authorize_exam_runs(self, authorized, bulk_authorize_for_exam_run_mock):
+    @patch('exams.tasks.authorize_for_latest_passed_course')
+    def test_authorize_exam_runs(self, authorized, authorize_for_latest_passed_course_mock):
         """Test authorize_exam_runs()"""
```

Comment:
could you update the test function name and the docstring?

Reply:
I didn't change the task name that is being tested. What would you suggest to rename it to?